src/pip/_internal/operations/freeze.py

```diff
@@ -17,17 +17,32 @@
 from pip._internal.utils.misc import (
     dist_is_editable, get_installed_distributions,
 )
+from pip._internal.utils.typing import MYPY_CHECK_RUNNING
+
+if MYPY_CHECK_RUNNING:
+    from typing import (  # noqa: F401
+        Iterator, Optional, List, Container, Set, Dict, Tuple, Iterable, Union
+    )
+    from pip._internal.cache import WheelCache  # noqa: F401
+    from pip._vendor.pkg_resources import (Distribution,  # noqa: F401
```

Comment:
Use the same indentation style here:

```suggestion
    from pip._vendor.pkg_resources import (  # noqa: F401
        Distribution, Requirement,
    )
```